packages/terra-tabs/examples/test-examples/Tabs/TabsTemplate.jsx

```diff
@@ -19,7 +19,7 @@ const defaultProps = {
 
 const TabsTemplate = props => (
   <Base locale={locale}>
-    <div style={{ backgroundColor: 'lightgrey', height: props.containerHeight }}>
+    <div style={{ height: props.containerHeight }}>
```

Comment:
this was causing issues with the themed examples (with transparent tabs) and doesn't seem to be needed for core when comparing my branch to master. Let me know if i'm missing something
http://engineering.cerner.com/terra-core/#/site/components/site/tabs/index
https://terra-core-deployed-pr-1529.herokuapp.com/#/site/components/site/tabs/index